packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-accept.tsx

```diff
@@ -1,16 +1,18 @@
 import * as React from 'react'
+import cx from 'classnames'
 import { TeamsProcessedSvgIconSpec } from '../types'
+import { teamsIconSlotClassNames } from '../index'
```

Comment:
We should move `teamsIconSlotClassNames` to as it produces circular imports.

See `Project Tests` in CI:
```
(!) Circular dependency: node_modules/@stardust-ui/react/dist/es/themes/teams/components/Icon/svg/index.js -> node_modules/@stardust-ui/react/dist/es/themes/teams/components/Icon/svg/icons/index.js -> node_modules/@stardust-ui/react/dist/es/themes/teams/components/Icon/svg/icons/undo.js -> node_modules/@stardust-ui/react/dist/es/themes/teams/components/Icon/svg/index.js
```